src/applications/hca/tests/00-all-fields.e2e.spec.js

```diff
@@ -154,13 +145,16 @@ module.exports = E2eHelpers.createE2eTest(client => {
     '/military-service/additional-information',
   );
 
-  if (process.env.BUILDTYPE !== ENVIRONMENTS.VAGOVPROD) {
-    // Military Service Documents Page.
+  // Military Service Documents Page.
+  client.waitForElementVisible('label[for="root_attachments"]', Timeouts.slow);
 
-    client.waitForElementVisible(
-      'label[for="root_attachments"]',
-      Timeouts.slow,
-    );
+  // This test is disabled in prod because I cannot get it to pass in Jenkins.
+  // In particular the test to check the value of
+  // #root_attachments_0_attachmentName fails because that element never appears
+  // after the value of the file input field is set
+  if (process.env.BUILDTYPE !== ENVIRONMENTS.VAGOVPROD) {
+    // Looks like there are issues with uploads in nightwatch and Selenium
+    // https://github.com/nightwatchjs/nightwatch/issues/890
     E2eHelpers.uploadTestFile(client, testData.data.testUploadFile);
```

Comment:
I tried skipping this helper doing the file attachment like we do in the pensions e2e test:
```js
client.setValue(
  'input[type="file"]',
  require("path").resolve(
    `${__dirname}/${testData.data.testUploadFile.fileName}`
  )
);
```